opt/scopeserver/dataserver/utils/loom_file_handler.py

```diff
@@ -34,12 +35,16 @@ def load_loom_file(self, partial_md5_hash: str, file_path: str, abs_file_path: s
             logger.warning(f"Deleting malformed loom {file_path}")
             return None
         return self.add_loom(
-            partial_md5_hash=partial_md5_hash,
+            file_hash=file_hash,
             file_path=file_path,
             abs_file_path=abs_file_path,
             loom_connection=loom_connection,
         )
 
+    @staticmethod
+    def get_file_hash(file_path: str):
+        return imohash.hashfile(file_path, hexdigest=True)
```

Comment:
I don't really understand what this was trying to achieve in the old version either @KrisDavie ?
I realise timestamps etc might change and you don't want to change the hash if that changes... what is the high-level goal you're trying to achieve? *And* what's the bug in the other project that this is supposed to fix?
```suggestion
        hash = hashlib.md5()
        with open(file_path, "rb") as f:
            buffer = f.read(1024) # or something...
            while len(buffer) > 0:
                hash.update(buffer)
                buffer = f.read(1024)
        return hash.hexdigest()
```